src/appleseed/renderer/kernel/rendering/masterrenderer.cpp

```diff
@@ -255,10 +260,21 @@ struct MasterRenderer::Impl
             m_shading_system->attribute("searchpath:shader", project_search_paths);
         }
 
+        // Initialize the shader compiler, if the OSL headers are found.
+        if (m_resource_search_paths.exist("stdosl.h"))
+        {
+            const APIString stdosl_path = m_resource_search_paths.qualify("stdosl.h");
+            RENDERER_LOG_INFO("found OSL headers in %s", stdosl_path.c_str());
```

Comment:
I know, I know, this rule is weird, and we'll fix it, but until then and for consistency's sake OSL should be spelled in lower case in log messages.

Reply:
Ok. At some point I'll change all log messages to use proper casing...